content/docs/trust/trust-manager/README.md

```diff
@@ -88,18 +91,27 @@ spec:
 - `inLine` - a manually specified string containing at least one certificate
 - `useDefaultCAs` - usually, a bundle of publicly trusted certificates
 
-These sources, along with the single currently supported target type (`configMap`)
-are documented in the trust-manager [API reference documentation](./api-reference.md).
+`ConfigMap` is the default target resource type, but trust-manager also supports `Secret` as targets
+(since trust-manager v0.7.0), but that feature has to be explicitly enabled on the controller.
+
+All sources and target options are documented in the trust-manager [API reference documentation](./api-reference.md).
 
 #### Targets
 
-All `Bundle` targets are written to `ConfigMap`s whose name matches that of the `Bundle`, and every
-target has a PEM-formatted bundle included.
+All `Bundle` targets are written to `ConfigMap`s (and/or `Secret`s) whose name matches that of the
+`Bundle`, and every target has a PEM-formatted bundle included.
+
+Users can also optionally choose to write JKS/PKCS#12 formatted binary trust store(s) to the target.
+JKS is supported since trust-manager v0.5.0, while v0.7.0 add support for PKCS#12.
 
-Users can also optionally - as of trust-manager v0.5.0 - choose to write a JKS formatted binary
-bundle to the target. We understand that most Java applications tend to require a password on JKS
-files (even though trust bundles don't contain secrets), so all trust-manager JKS bundles use the
-default password `changeit`.
+We understand that any binary trust store require a password, even though trust bundles don't contain
+secrets. Both JKS and PKCS#12 uses weak encryption primitives, so a trust store (or keystore) will NOT
+be protected by a password alone, and needs to be protected by additional measures.
+Please understand that trust-manager provides these binary trust store formats to support legacy
+applications which do not support other formats.
```

Comment:
suggestion:

> Please understand that trust-manager provides these binary trust store formats to support legacy
> applications which do not support other formats.

I don't think this is exclusively for legacy stuff. An app using JKS trust stores today probably doesn't have anything wrong with it - JKS is overcomplicated for the task for sure, but it'd work just fine and I'm sure it'll be supported basically forever. I'd remove these two lines personally.

Reply:
Agreed, removed.